src/interactive-config.coffee

```diff
@@ -69,14 +65,8 @@ interactiveConfig.prompt = (config = {}, callback) ->
     message: "Please enter Apiary API name"
     default: config['custom']?['apiaryApiName']
     when: (answers) ->
-      # it's not GA on the Apiary side yet
-      if process.env['APIARY_BETA']?
-        (
-          (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
-          (answers['apiary'] == true && answers['apiaryApiKey'] != '')
-        )
-      else
-        false
+      (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
+      (answers['apiary'] == true && answers['apiaryApiKey'] != '')
```

Comment:
I would love to, but no time today. The whole interactive config wizard would deserve facelift.